### ISSUE TITLE
Error: ADmad/SocialAuth.AuthController could not be found when posting from login to facebook action

### ISSUE THREAD

**Issue by JQEstrada** [ADmad/cakephp-social-auth#37]
Greetings,

I am sorry I didn't make myself clearer before, and this issue refers to the previously closed issue #36 .

The error is generated when clicking the login link generated by the readme instructions (postlink), and thus following the POST request:
Request URL: http://testface.ml/social-auth/login/facebook
Request Method: POST
Status Code: 404 Not Found

Hence the error
![error](https://user-images.githubusercontent.com/25327318/38951502-bfda2ee2-4340-11e8-8b68-2a8978c7e474.jpg)
 "Error: ADmad/SocialAuth.AuthController could not be found" (attached image)

**Comment by ADmad** [comment]
Don't open new issues for the same problem. You can post comments even on closed issues. Are you using the latest release?  What plugin version does `composer show` display?

**Comment by ADmad** [comment]
The error stack trace shows you are using an older buggy version, upgrade to latest version `0.3.2`. The method calls highlighted should be `$request->getParam()` instead of `$request->getQueryParams()`.

![screenshot-testface ml-2018-04-19-00-23-31](https://user-images.githubusercontent.com/142658/38952134-3db285a4-4368-11e8-88a1-282db66ae7da.png)

**Comment by JQEstrada** [comment]
I'm sorry, I am not used to the system.

About the version I should have mentioned it earlier, composer was reverting the installation of version 0.3.2 because if "conflicted with requirements or minimum-stability", even with the composer file set to "minimum-stability": "dev",
    "prefer-stable": true"

Therefore I manually added a 0.3.2 version of the files to the Cake PHP installation, and the getQueryParam was the only method I had to change, because it wasn't accepted by the framework (maybe related with the incompatibility?)

The plugin is otherwise working as expected, I didn't associate this error with the way it was installed. Is there any way to confirm if these are related, or to install it properly bypassing the compatibility issue?
